src/metabase/lib/convert.cljc

```diff
@@ -143,17 +168,20 @@
   "Coerce something to legacy MBQL (the version of MBQL understood by the query processor and Metabase Lib v1) if it's
   not already legacy MBQL."
   {:arglists '([x])}
-  lib.dispatch/dispatch-value)
+  lib.dispatch/dispatch-value
+  :hierarchy lib.hierarchy/hierarchy)
 
 (defn- lib-key? [x]
   (and (qualified-keyword? x)
        (= (namespace x) "lib")))
 
-(defn- disqualify [x]
-  (->> x
+(defn- disqualify
+  "Remove any keys starting with the `:lib/` namespace from map `m`."
+  [m]
```

Comment:
`m` is a more idiomatic variable name for something that is a map than `x`, makes this a little easier for me to remember what it's doing